src/matlab/reactormethods.cpp

```diff
@@ -48,6 +48,13 @@ void reactormethods(int nlhs, mxArray* plhs[],
         case 7:
             iok = reactor_setKineticsMgr(i, int(v));
             break;
+        case 8:
+            // This casting of double to bool is recommended by the MathWorks
```

Comment:
I'm not normally one to discourage comments, but I don't really understand what this comment clarifies. The cast to bool seems pretty straightforward to me (and I wouldn't be surprised if the compiler emitted a warning if you didn't)